lib/odata_duty/executor.rb

```diff
@@ -13,6 +13,7 @@ def self.create(**kwargs)
     def initialize(schema:, url:, context:, query_options:)
       @schema = schema
       @url = url
+      @base_url = schema.base_url.to_str
```

Comment:
storing the base url on the schema object allows the schema to be built not only in a controller but also in a background job, which is needed to respond to MCP requests